plutus-chain-index/src/Plutus/Contract/CardanoAPI.hs

```diff
@@ -184,7 +192,16 @@ toCardanoTxOut networkId (P.TxOut addr value datumHash) =
             <*> toCardanoTxOutDatumHash datumHash
 
 fromCardanoAddress :: C.AddressInEra era -> Either FromCardanoError P.Address
-fromCardanoAddress (C.AddressInEra C.ByronAddressInAnyEra _) = Left ByronAddressesNotSupported
+fromCardanoAddress (C.AddressInEra C.ByronAddressInAnyEra (C.ByronAddress address)) =
```

Comment:
Don't Byron addresses have a different length? I'm worried that this stops working when we convert the `P.Address` back to a `cardano-api` address. We can leave it in for now, but I'll try use the original `C.AddressInEra` in more places.

Reply:
So we need something in `P.Address` to say which era the address comes from? I think there would be a bunch of changes to be made, so I think a new PR would be welcomed for it. Feel free to assign me.

Reply:
Yeah probably. We basically need an exact copy of their address type